src/app/_pages/join/join.page.scss

```diff
@@ -0,0 +1,60 @@
+mat-grid-tile {
+    background: #0277bd;
+    border-radius: 5px;
+}
+
+button { 
+    position: absolute; 
```

Comment:
careful with using position absolute. absolute pulls the element out of the element render flow which means it could cover up other elements. You probably won't notice this unless you have enough groups to cover the page vertically. If this happens and the groups don't have a bottom margin sufficient enough to clear any possible button then they will end up intersecting the same space with it. The options to fix this include adding a margin larger than the button height/position so the groups can clear when scrolled all the way, or have the content and the button in their own containers and set the content container to expand vertically with flexbox so the button container is pushed to the bottom of the page (but could be scrolled off the page for long lists).